src/Caching/Hybrid/src/Internal/DefaultHybridCache.cs

```diff
@@ -0,0 +1,60 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Caching.Distributed;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.Extensions.Caching.Hybrid.Internal;
```

Comment:
Do we still need Internal namespaces now that we make the types themselves internal?

Reply:
@amcasey  happy to apply any convention here; doesn't impact the public API, so ... 🤷

Reply:
Question for @halter73 - I don't feel strongly either way.